velox/connectors/hive/FileHandle.cpp

```diff
@@ -43,8 +43,13 @@ std::shared_ptr<FileHandle> FileHandleGenerator::operator()(
   {
     MicrosecondTimer timer(&elapsedTimeUs);
     fileHandle = std::make_shared<FileHandle>();
-    fileHandle->file = filesystems::getFileSystem(filename, properties_)
-                           ->openFileForRead(filename);
+    if (properties_) {
+      fileHandle->file = filesystems::getFileSystem(filename, properties_)
+                             ->openFileForRead(filename);
+    } else {
+      fileHandle->file =
```

Comment:
Why we assume local file system access if properties_ is not set?

Reply:
The underlying assumption is that any remote storage would need non-null config. If properties are missing, then the local file system is used.

Reply:
I should be adding a check for the file scheme here for the local file system.